lib/js-api/src/api/content-rest-api/model/group.ts

```diff
@@ -18,6 +18,7 @@
 export interface Group {
     id?: string;
     displayName?: string;
+    description?: string;
```

Comment:
I think we need to update [Group.md](https://github.com/Alfresco/alfresco-ng2-components/blob/develop/lib/js-api/src/api/content-rest-api/docs/Group.md)

Reply:
done